### ISSUE TITLE
any way to generate music longer than 47 seconds？

### ISSUE THREAD

**Issue by lszhou0126** [Stability-AI/stable-audio-tools#154]
As the title described. Can continuation methods be used?

**Comment by kenkalang** [comment]
just change the 'sample_size' to a value longer than 47s on model_config and you should be able to do it

**Comment by lszhou0126** [comment]
> just change the 'sample_size' to a value longer than 47s on model_config and you should be able to do it

@kenkalang Thank you for your response. While the length has indeed increased, the audio quality seems to have deteriorated, and the conditions for seconds_start and seconds_total are not functioning as expected. The pre-trained model had a duration of 47 seconds, so it seems that a direct modification like this might not be appropriate.

**Comment by kenkalang** [comment]
yeah, you should also fine tune the pre trained model with your dataset if you want it to have better quality

**Comment by lszhou0126** [comment]
> yeah, you should also fine tune the pre trained model with your dataset if you want it to have better quality

If fine-tuning is to be performed, aside from DiT, does the Autoencoder part of the network also require fine-tuning?

**Comment by NZqian** [comment]
As the autoencoder is fully convolution-based, I think it does not need any fine-tuning

**Comment by kenkalang** [comment]
> As the autoencoder is fully convolution-based, I think it does not need any fine-tuning

yeah, just tune the DiT that's where the most impact happens

**Comment by zaptrem** [comment]
Look up the multi diffusion paper, should work fine with this to generate arbitrary length music.

**Comment by lszhou0126** [comment]
> Look up the multi diffusion paper, should work fine with this to generate arbitrary length music.

@zaptrem Are you referring to the "Fusing Diffusion Paths for Controlled Image Generation" paper when you mention multi diffusion? Is there any other application related to music?

**Comment by zaptrem** [comment]
> > Look up the multi diffusion paper, should work fine with this to generate arbitrary length music.
> 
> @zaptrem Are you referring to the "Fusing Diffusion Paths for Controlled Image Generation" paper when you mention multi diffusion? Is there any other application related to music?

Page 49 of Movie Gen they use it for their audio accompaniment model: https://ai.meta.com/static-resource/movie-gen-research-paper
